### DISCUSSION TITLE
我永远喜欢深海少女

### DISCUSSION THREAD

**Discussion by kont9** [share121/inter-knot#511] [其他]
![120676842_p0_master1200](https://github.com/user-attachments/assets/1f9d1389-8a10-40a6-aa84-2cc1b0c542ec)
###  **_能不能给世界第一公主殿下打callQAQ_**
https://www.bilibili.com/video/BV1Hx411c7Gh/?share_source=copy_web&vd_source=1a59e584a9fa0c269ede5d69baab5263
https://www.bilibili.com/video/BV1s44y1b7Tw/?share_source=copy_web&vd_source=1a59e584a9fa0c269ede5d69baab5263
 https://www.bilibili.com/video/BV1Az4y1V7w2/?share_source=copy_web&vd_source=1a59e584a9fa0c269ede5d69baab5263
第一个是本家︿(￣︶￣)︿，第二个是高清版本φ(>ω<*) ，第三个是Jiafei的产品，第三个没有做好心理准备的情况下建议不要点开(；д；)。

封面图https://www.pixiv.net/artworks/120676842 侵删

**Comment by leonwangls** [comment]
[肯定]:你的厨力我认可了

**Comment by MiaoJiSir** [comment]
[警报]绳网出现不明链接，希望各位绳匠前去探查！探查后可到喵吉长官处领取奖励！

**Comment by lllliufeng** [comment]
您即将离开绳网，去往https://www.bilibili.com/
请注意您的账号和财产安全
